src/open_inwoner/scss/components/SearchResults/SearchResults.scss

```
@@ -28,38 +22,43 @@
     letter-spacing: 0;
     line-height: var(--font-line-height-body);
     text-decoration: none;
-  }
 
-  &__item-title {
-    font-size: var(--utrecht-heading-3-font-size);
-    line-height: var(--utrecht-heading-3-line-height);
-    letter-spacing: 0;
-    margin: 0 0 6px;
-  }
+    .utrecht-heading-3.search-results__item-title {
+      margin: 0 0 var(--spacing-large) 0;
+    }
 
-  &__item-info-container {
-    display: grid;
-    grid-template-columns: 1fr auto;
-    gap: var(--spacing-large);
-    align-items: end;
+    .utrecht-paragraph {
+      margin: 0;
+    }
   }
 
-  &__item-intro {
-    color: var(--color-black);
-    font-family: var(--font-family-body);
-    font-size: var(--font-size-body);
-    letter-spacing: 0;
-    line-height: var(--font-line-height-body);
-    margin: 0;
+  .link {
+    padding-top: var(--card-spacing);
   }
 
-  &__item-type {
-    color: var(--color-black);
-    font-family: var(--font-family-body);
-    font-size: var(--font-size-body);
-    letter-spacing: 0;
-    line-height: var(--font-line-height-body);
-    text-align: right;
-    margin: 0;
+  &--none {
```

```suggestion
  &--no-results {
```
More clear what this modifier does.
I was confused bc the classname is `--none`, but the display property is flex (not none).

---

Good idea!